common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -96,16 +96,17 @@ case class VideoEmbedCleaner(article: Article) extends HtmlCleaner {
       element.getElementsByClass("gu-video").isEmpty
     }.foreach { element: Element =>
       val canonicalUrl = element.attr("data-canonical-url")
+      element.children().lift(0).map(child => {
```

Comment:
Nice use of `lift` 

Would marginally prefer `.lift(0).map { ... }` notation for consistency though